src/Core/functions.php

```diff
@@ -61,9 +61,6 @@ function h($text, bool $double = true, ?string $charset = null)
         static $defaultCharset = false;
         if ($defaultCharset === false) {
             $defaultCharset = mb_internal_encoding();
-            if ($defaultCharset === null) {
```

Comment:
If there is no encoding set, this method returns null and I think we should fallback to `utf-8` still.